src/main/java/com/onarandombox/MultiverseCore/commands/CreateCommand.java

```diff
@@ -98,7 +96,7 @@ public void onCreateCommand(CommandIssuer issuer,
                                 @Description("") //TODO
                                 String[] flags
     ) {
-        ParsedFlags parsedFlags = this.plugin.getCommandManager().getFlagsManager().parse("mvcreate", flags);
+        ParsedCommandFlags parsedFlags = this.plugin.getCommandManager().getFlagsManager().parse("mvcreate", flags);
```

Comment:
If `MultiverseCommand` has a `registerFlagGroup` method, maybe it could store the group name and then just have a `parseFlags(flags)` call here.

Reply:
the issue is with auto complete, there isnt a way to get the flaggroup from ACF's autocomplete handler, currently this is what is needed. There also isnt a way to get the command class from the CommandCompletionContext (unless we do reflection which is what i wanted to avoid).

https://github.com/Multiverse/Multiverse-Core/blob/85f92bbbc5c2f09362cffcfc63ae6732c7762695/src/main/java/com/onarandombox/MultiverseCore/commandtools/MVCommandCompletions.java#L19-L23

Reply:
ohhh wait, nvm i get what you mean, just a shortcut method in multiverse commands